client/src/components/layout/ReviewTile.js

```diff
@@ -1,12 +1,16 @@
 import React from "react";
+import ReviewEditButton from "./ReviewEditButton";
+import ReviewDeleteButton from "./ReviewDeleteButton"
 
-const ReviewTile = ({review}) => {
+const ReviewTile = ({ review, user, getPlanet }) => {
 
   return(
-    <>
+    <div className = "review">
```

Comment:
convention has always been to avoid spaces before and after the `-` when making an HTML attribute like className. Please update